wolfssl/openssl/ssl.h

```diff
@@ -726,6 +731,10 @@ wolfSSL_X509_STORE_set_verify_cb((WOLFSSL_X509_STORE *)(s), (WOLFSSL_X509_STORE_
 #define X509_OBJECT_get0_X509           wolfSSL_X509_OBJECT_get0_X509
 #define X509_OBJECT_get0_X509_CRL       wolfSSL_X509_OBJECT_get0_X509_CRL
 
+#define X509_REVOKED_get_serial_number   wolfSSL_X509_REVOKED_get_serial_number
```

Comment:
I don't think X509_REVOKED_get_serial_number exists in OpenSSL, as far as I can tell looking at their source code.